hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManagerStarter.java

```diff
@@ -98,6 +98,28 @@ public void initOm()
     }
   }
 
+
+  /**
+   * This function implements a sub-command to allow the OM to be
+   * "prepared for upgrade".
+   */
+  @CommandLine.Command(name = "--prepareForUpgrade",
+      aliases = {"--prepareForDowngrade", "--flushTransactions"},
```

Comment:
Will prepareForUpgrade command send multiple OMs simultaneously here? Or we should trigger prepareForUpgrade command for each OM service.

Reply:
This command should be issued when the OM is already stopped before the upgrade of software bits. This is a command that starts up the OM code in a special way, with that it can start up only the current local OM, as I understand.

Reply:
+1 to @fapifta's reply.